.license-sh.json

```diff
@@ -14,5 +8,11 @@
     "Python Software Foundation License",
     "Apache 2",
     "MIT"
-  ]
+  ],
+  "ignored_packages": {
+    "pipenv": [
+      "aiohttp:-:3.6.2",
```

Comment:
What is the delimiter here? 
Shouldn't we use the delimiter used in standard `requirements.txt`?

Reply:
https://packaging.python.org/tutorials/installing-packages/#installing-from-pypi

Reply:
It is different for every language... I have used the one that we are using for node id. It is visible in the json output

Reply:
@S4n60w3n exactly. I'd like to keep them relevant to a local package manager. It can be useful to have it. There is no such a thing as `aiohttpz:-:3.6.2`

Reply:
@jvorcak And do you want to handle all cases? 
* Pipenv
`SomeProject`
`SomeProject==1.4`
`SomeProject>=1,<2`
`SomeProject~=1.4.2`
* NPM/Yarn
`SomeProject`
`SomeProject@1.4`
`SomeProject@^1.4`

Or do you mean just pipenv version for everyone and only specific version?
`SomeProject==1.4`

Reply:
@S4n60w3n depends. I think for an initial version, it's OK to just cover exact versions. Supporting various expressions per language can be a feature later.

Reply:
Let's stick with a fixed version. I feel like supporting semver inspires bad practices of usage as for the ignore feature.